### ISSUE TITLE
[Event Request] Codeunit 21 OnAfterAssignDimensions

### ISSUE THREAD

**Issue by richardrobberse** [Microsoft/AL#3126]
I would like to have an event publisher in codeunit 21, function CheckDimensions so that we can assign our own dimensions to the TableID and No array.

The signature should look like this: 
OnAfterAssignDimensions(ItemJnlLine: Record "Item Journal Line", TableID: Array [10] of Integer,No: Array [10] of Code[20].

**Comment by thpeder** [comment]
Reply from the related team:

You should not change data in COD11, this is for checks only.

**Comment by richardrobberse** [comment]
Makes sense but then Microsoft is doing it in the wrong place and I'm forced to do it in the wrong place as well ;)
